docs/wiki/NOS3_cFS_Development.md

```diff
@@ -0,0 +1,114 @@
+
+![Flight versus Simulation](./_static/NOS3-Flight-vs-Simulation.png)
+
+## Connecting cFS to NOS3
+
+In order to use NOS3 with cFS, modifications are required to the open-source release. The recommended method for using NOS3 is described in the NOS3 User’s Guide, in which these modifications have already been made. If not using the cFS included with the NOS3 release, it is recommended to use the CMAKE build system, as the legacy build is not currently supported. The necessary changes are described below, where `proj` is the cFS directory being integrated.
+
+1. Edit the `targets.cmake` file in the `fsw/nos3_defs` folder to include the list of applications to be built. Set the target name and system as shown below.
+```cmake
+SET(TGT1_NAME linux)
+SET(TGT1_SYSTEM linux)
+```
+2. Edit the `toolchain-linux.cmake` file from the `fsw/nos3_defs` directory.
+3. Edit the `nos-linux` PSP from the `fsw/psp/fsw` directory.
+4. Add to the `components` directory as needed.
+5. Create `fsw/apps/hwlib` or edit the `fsw/apps/hwlib` directory.
+   1. The `CMakeLists.txt` file in `fsw/apps/hwlib` provides a good example of how to include driver source code as described below.
+   2. Add a `sim` folder to this directory to store the NOS3 drivers for I/O. (See below for an example driver)
+
+## NOS3 Drivers and Other FSW
+
+It is possible to connect NOS3 to FSWs other than cFS, although this has not been extensively tested. The two main requirements are the availability of source code for the I/O drivers, and the ability to compile/run on Linux. If these two conditions are met, the drivers for the target hardware can be swapped for NOS3 drivers as described in previous sections.
+
+### Writing a NOS3 Driver
+
+The NOS3 source is the best resource for examples to aid in writing a new NOS3 driver. The UART driver and STF-1 NAV (navigation) application will be used in the example described in this section. For this example, the NAV application is written for cFS and located in the `novatel_oem615` component folder, but this application could just as easily be any other FSW source file.
+
+#### Application and Hardware Library
+
+The application that is communicating with hardware will require the I/O calls to be implemented exactly as provided by the OBC manufacturer. The NAV application makes certain calls to a Novatel GPS over the UART from the OBC. Not all of the GPS functionality is necessary to be exercised by the NAV application, so the low level calls to the UART are wrapped in functions in the hardware library, and the NAV app includes this library. As an example, the NAV application will be commanded to get the current Position/Velocity/Time reading, and will make the call `NAV_ReadAvailableData` as seen in the following code excerpt. Notice the include statement for the hardware library.
+
+```c
```

Comment:
The 2 code snippets below no longer exist in the code.  Replace them with:
novatel_oem615_app.c, NOVATEL_OEM615_ChildProcessRequestData
novatel_oem615_device.c, NOVATEL_OEM615_ChildProcessReadData